lib/reducers/create/error.js

```diff
@@ -20,4 +16,22 @@ function error(config, current, addedRecord) {
   return reject(current, predicate);
 }
 
+function errorMap(config, current, addedRecord) {
+  var key = config.key;
+  return omit(current, addedRecord[key]);
+}
+
+function error(config, current, addedRecord) {
+  var reducerName = 'createError';
+  assertNotArray(config, reducerName, addedRecord);
+
+  addedRecord = common(config, current, addedRecord, reducerName);
+
+  if (config.store === constants.STORE_MAP) {
+    return errorMap(config, current, addedRecord);
+  } else {
+    return errorList(config, current, addedRecord);
+  }
+}
+
```

Comment:
@sporto What do you think about maybe having some kind of storage adapter layer here instead of having to add if statements in all of the reducers?

Something like
```
var store = require('../store');

function error(config, current, addedRecord) {
  // ....

  return store[config.store].remove(config, current, addedRecord)
}

```